src/snappy_device_agents/devices/muxpi/muxpi.py

```diff
@@ -236,13 +236,13 @@ def get_image_type(self):
         def check_path(dir):
             self._run_control("test -e {}".format(dir))
 
-        # First check if this is a limerick image
+        # First check if this is an oem image
         try:
             disk_info_path = self.mount_point / "writable/.disk/info"
-            self._run_control(f"grep limerick {disk_info_path}")
-            return "limerick"
+            self._run_control(f"grep -E 'limerick|erlangen' {disk_info_path}")
```

Comment:
This will result in a continually growing list of projects that need to be grepped for.

I mechanism should specified and agreed upon that does not require continual changes in to the future modification

Reply:
This is how the build stamp looks like for all the images built from CE Jenkins:
```
Sun, 14 May 2023 19:08:32 +0000
Jenkins
iot-erlangen-classic-server-2204-X00-20230514-85
```

It'll always be a `Jenkins` string in the build stamp file `/.disk/info`, do you think it'll be better if we use this as the keyword so with the change we can really say this image type supports all the image built from CE Jenkins.

Reply:
Pushed new commit to change the approach of checking image_type, now checking the buildstamp inside of "writable/.disk/info" following the naming convention defined in [IoT: CI/CD Integration process document](https://docs.google.com/document/d/1PXMcUHiguli5y6AY1HJ-6GDzmZVyiTiY8bjbE94C6H0/edit#heading=h.8m5xai2s8gbe)

In the build script used to generate buildstamp for CE OEM IoT project images, it's using the jenkins job name defined following process doc mentioned above as part of its buildstamp string: 

```
cat << EOF | sudo tee buildstamp
$(date -R)
Jenkins
$job_name-$milestone-$timestamp-$build_id
EOF
```